### PR TITLE
feat: #1285 pass user guid from frontend, update backend schema

### PR DESCRIPTION
**Pull request by MCatherine1994**
refs: #1285

- Regenerate client code
- Frontend store user_guid in the form data after the  user verification, and pass it when make api calls to create user role assignment, create delegated admin and create application admin
- Backend update the schema for creating user role assignment, creating delegated admin and creating application admin to accept user_guid
- Update testing data to include user_guid

### CODE REVIEW

File: server/admin_management/tests/constants.py [hunk 1]
```diff
@@ -36,6 +38,8 @@
     }
 )
 
+TEST_USER_GUID_IDIR = os.environ.get("USER_GUID_IDIR") or ""
```
Comment (MCatherine1994) [inline server/admin_management/tests/constants.py:41]: Hi Ian, I think maybe we can discuss a little bit about how we'll use the user_guid of IDIR user when run tests. For now, I just put a placeholder, and it's just empty. But after we add the user verification in backend, either we provide the user_guid through environment variable from GitHub secret, or we skip the IDIR verification when run tests.

Comment (MCatherine1994) [reply server/admin_management/tests/constants.py:41]: discussed that we might don't need to do user validation immediately, so will worry about it later.

File: server/backend/api/app/schemas.py [hunk 2]
```diff
@@ -283,6 +284,7 @@ class IdimProxyIdirInfo(BaseModel):
     # property returned from Idim-Proxy search of this form (not snake case)
     found: bool
     userId: Annotated[str, StringConstraints(max_length=20)]
+    guid: Optional[Annotated[str, StringConstraints(max_length=32)]] = None
```
[the remark this answers is not in the document]
Comment (ianliuwk1019) [reply server/backend/api/app/schemas.py:287]: Never mind. We have discussed.

**Review by ianliuwk1019 — COMMENTED** [review]
Thank you Catherine! It's good for me, just very few small wondering.

File: frontend/src/components/grantaccess/GrantApplicationAdmin.vue [hunk 3]
```diff
@@ -39,8 +40,12 @@ const userIdChange = (userId: string) => {
 };
```
[the remark this answers is not in the document]
Comment (MCatherine1994) [reply frontend/src/components/grantaccess/GrantApplicationAdmin.vue:40]: yes, thanks Ian, I missed that

**Review by ianliuwk1019 — APPROVED** [review]
Thanks Catherine!

**Pull request merged**